modules/configuration-annotation-processor/src/integrationTest/java/org/apache/ignite/internal/configuration/processor/AbstractProcessorTest.java

```diff
@@ -133,7 +132,7 @@ public BatchCompilation(List<ClassName> schemaClasses, Compilation compilation)
             generatedSources = compilation.generatedSourceFiles();
 
             generatedClasses = generatedSources.stream()
-                    .collect(Collectors.toMap(object -> fromGeneratedFilePath(object.getName()), Functions.identity()));
+                    .collect(Collectors.toMap(object -> fromGeneratedFilePath(object.getName()), o -> o));
```

Comment:
You could use `Function.identity()` from `import java.util.function.Function;` package.